### ISSUE TITLE
java.util.Optional<java.time.OffsetDateTime> disappears from request params

### ISSUE THREAD

**Issue by bohdan-petrenko** [springfox/springfox#2138]
java.util.Optional<java.time.OffsetDateTime> disappears from request params after update from 2.6.1 to 2.7.0. Optional don't work without this line `.genericModelSubstitutes(Optional.class)` in 2.7.0 at all

Swagger configuration:

```java
@Configuration
@EnableSwagger2
public class SwaggerConfig {
    @Bean
    public Docket productApi() {
        return new Docket(DocumentationType.SWAGGER_2)
                .select()
                .apis(RequestHandlerSelectors.basePackage("com.ipan.tofy.controller"))
                .paths(regex("/.*"))
                .build()
                .genericModelSubstitutes(Optional.class)
                .apiInfo(...);
    }
}
```

Controller code:

```java
@RestController
@RequestMapping("users")
public class UserController {
    @JsonView(JsonViews.GetUserView.class)
    @PreAuthorize("hasAuthority('USERS_READ')")
    @GetMapping("find")
    public List<User> find(@RequestParam Optional<Integer> id,
                           @RequestParam Optional<String> email,
                           @ApiParam(value = "Phone example: 380977777777", example="380977777777")
                           @RequestParam Optional<String> phone,
                           @ApiParam(value = "Unix time in milliseconds or ISO datetime")
                           @RequestParam Optional<OffsetDateTime> since,
                           @RequestParam Optional<Date> sinceD,
                           @RequestParam OffsetDateTime sinceO,
                           @RequestParam List<OffsetDateTime> sinceArr) {

    } 
}
```

Screen with 2.6.1

![sw26](https://user-images.githubusercontent.com/12713697/33777204-9278338c-dc4c-11e7-8f4b-de88dd990ff1.JPG)

Screen with 2.7.0

![sw270](https://user-images.githubusercontent.com/12713697/33777314-fb923e94-dc4c-11e7-8b85-2248d1d74707.JPG)

Is it possible to fix it somehow in my project, because I need to use 2.7.0 version?

**Comment by dilipkrish** [comment]
Closing as a duplicate of #2133. 

Would be awesome if you could verify this with `2.7.1-SNAPSHOT`

**Comment by bohdan-petrenko** [comment]
Yes, sure. But in my project I'm using maven, so for verification I need a link to a repository where `2.7.1-SNAPSHOT` version located.

**Comment by bohdan-petrenko** [comment]
Yes, this bug fixed in `2.7.1-SNAPSHOT`

**Comment by dilipkrish** [comment]
Thanks for confirming @bohdan-petrenko
